Steps4Impact/App/Navigation.swift

```diff
@@ -62,8 +62,15 @@ class Navigation: UITabBarController {
 
     // FIXME(compnerd) enumerate the notifications
     self.notifications.tabBarItem.badgeValue = "0"
+    AppController.shared.askForPushNotificationPermissions()
```

Comment:
I wouldn't ask for permission right as app boots. Maybe after login or first time they go to the notification tab?